.envrc

```diff
@@ -0,0 +1,4 @@
+export VIRTUAL_ENV=."venv"
```

Comment:
I saw dev containers script used `.venv` name so made it consistent - happy to change it the other way to `venv` if you'd rather.

Reply:
I do most of my dev work out of vscode and vscode creates the .venv dir

Reply:
AHH sorry this one is wrong, yeah feels like with the leading . Is the normal one. Will update this and remove the other change when I get home